lib/srv/authhandlers.go

```diff
@@ -716,3 +699,27 @@ func (h *AuthHandlers) authorityForCert(caType types.CertAuthType, key ssh.Publi
 func (h *AuthHandlers) isProxy() bool {
 	return h.c.Component == teleport.ComponentProxy
 }
+
+// MakeAuthAttemptFailureEvent creates an auth attempt failure event using
+// provided info.
+func MakeAuthAttemptFailureEvent(conn ssh.ConnMetadata, ident *sshca.Identity, err error) *apievents.AuthAttempt {
```

Comment:
Does this need to be exported, or is it only used in `lib/srv`?

Are there other places where this is missing?

Reply:
it's used below in lib/srv/git/forward.go